ci/ci-test.sh

```diff
@@ -97,6 +97,10 @@ sudo vgcreate foreign_lvmvg "${foreign_disk}" --config="${FOREIGN_LVM_CONFIG}"
 sudo modprobe dm-snapshot
 sudo modprobe dm_thin_pool
 
+# Set the configuration for thin pool autoextend in lvm.conf
+sudo sed -i '/^[^#]*thin_pool_autoextend_threshold/ s/= .*/= 50/' /etc/lvm/lvm.conf
```

Comment:
Shouldn't this be part of the particular test itself?

Reply:
We just need this configured once, and it's simpler to do this as part of test setup, just like in reality users have to manually pre-configure this.